src/psyclone/tests/domain/lfric/lfric_loop_bounds_test.py

```diff
@@ -32,15 +32,15 @@
 # POSSIBILITY OF SUCH DAMAGE.
 # -----------------------------------------------------------------------------
 # Author: A. R. Porter, STFC Daresbury Lab
-# Modified: L. Turner, Met Office
+# Modified: L. Turner and O. Brunt, Met Office
 
 ''' This module contains pytest tests for the LFRicLoopBounds collection
 class. '''
 
 from __future__ import absolute_import
```

Comment:
Please rm this line (it was needed back when we supported both Python 2 and Python 3 and we're gradually removing them now).